src/components/icons/Icon.jsx

```diff
@@ -488,6 +488,18 @@ export type IconPropsType =
                   </Button>
       */
       tagType?: IconTagType,
+      /**
+       * An accessible, short-text description; defaults to `type`
+       */
+      title?: string,
+      /**
+       * An accessible, long-text description
+       */
+      description?: string,
+      /**
+       * Provides semantic meaning to content, defaults to "img"
+       */
+      role?: string,
```

Comment:
@katarzynatobis can it be different than img?

Reply:
If title and description are optional, then maybe we should set default role as 'presentation'? Otherwise accessibility checks will always throw warnings as title and description may not always always provided

Reply:
My understanding is:
- if we allow only `presentation` and `img` in role, let's reflect that in type of role - I think that default to `img` is a good way.
- what is the benefit of `presentation` role over `aria-hidden` in the context of our icon component? This would be alternate way allowing aria-hidden to be passed down to the component and having role always set to `img` without `role` prop being exposed

Reply:
Correct me if I am wrong but role presentation might not be the thing we want to have in here. It will just remove the native semantic meaning (which is graphical-content in our case) but the screen reader will still see svg nodes. I think that role should be always set to `img` and we should allow passing aria-hidden. wdyt?

Reply:
I have same thoughts after reading those materials @coderitual linked. It might be unnecessary to expose role as prop, since we know icon acts like an img. Aria hidden should do the job .

Reply:
I wasn't clear enough, sorry. `presentation` was only an example - based on w3, svg can [have any role](https://www.w3.org/TR/html-aria/#el-svg). But I agree with you - in that case, only `img` makes sense.

`aria-hidden` doesn't have to be passed to `svg`, it can be used on wrapper element. What's more, on wrapper it works for custom icon as well, so the icon api is more consistent and that's why I decided to add `aria-label`.